Sources/tart/Credentials/DockerConfigCredentialsProvider.swift

```diff
@@ -41,13 +41,17 @@ class DockerConfigCredentialsProvider: CredentialsProvider {
 
     process.waitUntilExit()
 
+    let outputData = try outPipe.fileHandleForReading.readToEnd()
     if !(process.terminationReason == .exit && process.terminationStatus == 0) {
+      let outputText = outputData == nil ? "" : String(decoding: outputData!, as: UTF8.self)
+      print(outputText)
```

Comment:
Perhaps we could simplify this down to:

```suggestion
      if let outputData = outputData {
        print(String(decoding: outputData, as: UTF8.self))
      }
```

Haven't checked this against compiler, though.